.github/workflows/update-flake.yml

```
@@ -41,22 +41,28 @@ jobs:
             email "$id+$name@users.noreply.github.com" \
             >>"$GITHUB_OUTPUT"
 
-      - uses: DeterminateSystems/update-flake-lock@v26
+      - name: setup git
+        env:
+          name: ${{ steps.user-info.outputs.name }}
+          email: ${{ steps.user-info.outputs.email }}
+        run: |
+          git config --global user.name $name
```

Always quote bash variable expansion, else they get split as separate args if they contain spaces.

Even if you know the value has no spaces, it's still a good habit to get into and also prevents bugs if the value ever changes.

```suggestion
          git config --global user.name "$name"
```